packages/react/README.md

```diff
@@ -72,15 +72,15 @@ the default way of interacting with Mintbase Wallet is using the MintbaseWalletC
 
 ```typescript
 import "@near-wallet-selector/modal-ui/styles.css";
-import { MintbaseWalletContextProvider } from  '@mintbase-js/react'
```

Comment:
Seems like this should just be called `WalletContextProvider` - independent of the name. If it were, then we wouldn't have had to change anything here.